### DISCUSSION TITLE
Feature Request: Add Event for Controls Show/Hide

### DISCUSSION THREAD

**Discussion by wigz** [vimeo/player.js#674] [Ideas]
An event that fires when controls are shown/hidden would be very useful for building custom controls that go alongside the standard Vimeo controls. For example, I want to add a large, easy to use unmute button to an autoplay player and have it fade in and out with the standard controls. It is difficult to mimic the functionality of the standard Vimeo controls because you can't handle `mousemove` over/in a sandboxed iframe.

`player.on('uihide', function() { ... });`
`player.on('uishow', function() { ... });`

**Comment by fisherinnovation** [comment]
Thanks for the feedback. We will review this internally and get back to you.

**Comment by Nabelops** [comment]
I know this is quite an old feature request but I would like to this as well.
